app/services/organizations/update_service.rb

```diff
@@ -58,10 +57,9 @@ def update_from_api(params:)
         organization.invoice_footer = billing[:invoice_footer] if billing.key?(:invoice_footer)
         organization.vat_rate = billing[:vat_rate] if billing.key?(:vat_rate)
 
-        # TODO(:grace_period): Grace period update is turned off for now
-        # if billing.key?(:invoice_grace_period)
-        #   Organizations::UpdateInvoiceGracePeriodService.call(organization:, grace_period: billing[:invoice_grace_period])
-        # end
+        if License.premium? && billing.key?(:invoice_grace_period)
+          Organizations::UpdateInvoiceGracePeriodService.call(organization:, grace_period: billing[:invoice_grace_period])
```

Comment:
Layout/LineLength: Line is too long. [122/120] (https://rubystyle.guide#max-line-length)